wordpress-mobile/WordPress-iOS
# Quick Start: add three guided tours

Refs #9447

This PR adds four of the guided tours:
- view site
- choose a theme
- customize

![first-three-tours](https://user-images.githubusercontent.com/517257/46781024-71d94e80-ccd4-11e8-8494-436f2bdf8427.gif)

To test:
- create a new site in the app (or ask me for a code snippet to fake this) and when asked if you want to use the quick start checklist tap "Yes"
- go to the checklist and tap through the three guided tours listed above
- make sure they work ok
